packages/svelte2tsx/test/sourcemaps/howto.md

```diff
@@ -0,0 +1,21 @@
+# SourceMapping tests
```

Comment:
I suggest to rename this file to `readme.md` . This has the benefit that it's rendered on Github when opening that folder.